README.md

```diff
@@ -1,2 +1,23 @@
 # comp110-worksheet-C
-Base repository for COMP110 worksheet C
+# Answers
+
+(a) Checks each item in the list agaisnt all other items in the list, if the list position is not the same but them item itself is the same, returns true. (checks if an item is duplicated but in different position)
+
+(b) Because both for loops are linear O(n) we multiple the compond algorithms making O(n^2)
+
+(c) Each iteration of i has already been checked against all positions. i.e 1st position is checked against all, now the 2nd position doesn't need to be checked agaisnt 1st position
+
+(d) Each loop of i we have to check one less, at the middle of the loop we are checking vs about half the list, and at the end only checking vs itself makings the j loops quicker and quicker
```

Comment:
✔️ although could be more specific about why this comes out to approximately twice as fast (e.g. the "average" case is only checking half the list)